MetaMorpheusSetup/Product.wxs

```diff
@@ -257,15 +251,30 @@
       <Component Id="src_System.Xml.XPath.XDocument.dll" Guid="70f63556-f331-4c08-ac9f-3515dacbf0e9">
         <File Id="src_System.Xml.XPath.XDocument.dll" Name="System.Xml.XPath.XDocument.dll" Source="$(var.GUI_TargetDir)System.Xml.XPath.XDocument.dll" />
       </Component>
-      <Component Id="src_OxyPlot.dll">
+      <Component Id="src_OxyPlot.dll" Guid="1B25D408-13DB-4A9E-A49E-B0A7FC8C05A6">
```

Comment:
Something changed here. Is this change necessary?

Reply:
https://wixtoolset.org/documentation/manual/v3/howtos/general/generate_guids.html

I don't fully understand it, but it seems like each of these components is supposed to have a GUID for Windows installer reasons. Some of these components didn't have one - I corrected that in this PR